test/unit/views/MessagesSpec.scala

```diff
@@ -39,7 +39,8 @@ class MessagesSpec extends PlaySpec with Injector {
     "cds.subscription-details.tab.title.data"
   )
 
-  def ignoreKey(key: String): Boolean = key.startsWith("cds.country.") || sameTranslation.contains(key)
+  def ignoreKey(key: String): Boolean =
```

Comment:
revert this since we have welsh translations now

Reply:
Done, Cheers!